src/rv/comm/rcssserver/GameState.java

```diff
@@ -335,6 +372,29 @@ public void parse(SExp exp, WorldModel world) {
                     scoreRight = Integer.parseInt(atoms[1]);
                     playStateChanges++;
                     break;
+                case FOUL:
```

Comment:
Looks like `FOUL` is the only expression that doesn't directly lead to one the `GameStateChangeListener`s being invoked. Not sure it really fits any of the existing ones though, and it seems like we don't need that to render them?